build.gradle

```diff
@@ -0,0 +1,93 @@
+import com.github.spotbugs.SpotBugsTask
+
+plugins {
+  id "com.github.hierynomus.license" version "0.14.0" apply false
+  id 'com.github.johnrengelman.shadow' version '2.0.1' apply false
+  id "com.google.protobuf" version "0.8.3" apply false
+  id "com.github.spotbugs" version "1.6.0" apply false
+}
+
+allprojects {
+  apply plugin: 'idea'
+  group = 'cz.seznam.euphoria'
+  version = '0.8-SNAPSHOT'
+}
+
+subprojects {
+
+  apply plugin: 'java'
+  apply plugin: 'com.github.hierynomus.license'
+  apply plugin: 'com.github.spotbugs'
+  apply from: "${rootProject.projectDir}/gradle/versions.gradle"
+
+  license {
+    include '**/*.java'
+    header = rootProject.file('HEADER')
+    strictCheck = true
+    ext.year = Calendar.getInstance().get(Calendar.YEAR)
+    ext.name = 'Seznam.cz, a.s.'
+  }
+
+  repositories {
+    mavenCentral()
+  }
+
+  dependencies {
+    compileOnly "org.projectlombok:lombok:${lombokVersion}"
+    testCompile "junit:junit:${junitVersion}"
+    testCompile "org.mockito:mockito-core:${mockitoVersion}"
+    testCompile "org.slf4j:slf4j-simple:${slf4jVersion}"
+  }
+
+  test {
+    testLogging {
+      events 'skipped', 'failed'
+    }
+  }
+
+  tasks.withType(SpotBugsTask) {
+    // ~ TODO: do not ignore failures and decrease report level
```

Comment:
This comment look serious and we should fix it before merging.

Reply:
it should be part of another pull request, so the reported issues will become more visible.